appeals/web/src/server/views/appeals/components/status-tag.njk

```diff
@@ -9,7 +9,7 @@
 	{%- switch (status | string | lower) -%}
 		{%- case 'ready_to_start' -%}
 			{%- set statusTagColor = 'turquoise' -%}
-		{%- case 'lpa_questionnaire_due' -%}
```

Comment:
I think we should keep this until we align the API, otherwise will fail as real appeals will still have `lpa_questionnaire_due`